build.gradle

```
@@ -46,20 +46,20 @@ dependencies {
     include "com.apollographql.apollo:apollo-normalized-cache-jvm:2.5.14"
     include "com.apollographql.apollo:apollo-runtime:2.5.14"
     include "com.benasher44:uuid-jvm:0.2.0"
-    include "com.bucket4j:bucket4j_jdk8-core:8.6.0"
-    include "com.fasterxml.jackson.core:jackson-annotations:2.15.3"
-    include "com.fasterxml.jackson.core:jackson-core:2.15.3"
-    include "com.fasterxml.jackson.core:jackson-databind:2.15.3"
-    include "com.fasterxml.jackson.datatype:jackson-datatype-jsr310:2.15.3"
-    include "com.fasterxml.jackson:jackson-bom:2.15.3"
+    include "com.bucket4j:bucket4j_jdk8-core:8.7.0"
+    include "com.fasterxml.jackson.core:jackson-annotations:2.16.1"
+    include "com.fasterxml.jackson.core:jackson-core:2.16.1"
+    include "com.fasterxml.jackson.core:jackson-databind:2.16.1"
+    include "com.fasterxml.jackson.datatype:jackson-datatype-jsr310:2.16.1"
+    include "com.fasterxml.jackson:jackson-bom:2.16.1"
     include "com.github.ben-manes.caffeine:caffeine:2.9.3"
-    include "com.github.philippheuer.credentialmanager:credentialmanager:0.2.2"
+    include "com.github.philippheuer.credentialmanager:credentialmanager:0.3.0"
     include "com.github.philippheuer.events4j:events4j-api:0.12.1"
     include "com.github.philippheuer.events4j:events4j-bom:0.12.1"
     include "com.github.philippheuer.events4j:events4j-core:0.12.1"
     include "com.github.philippheuer.events4j:events4j-handler-simple:0.12.1"
     include "com.github.tony19:named-regexp:0.2.8"
-    include "com.google.errorprone:error_prone_annotations:2.10.0"
+    //include "com.google.errorprone:error_prone_annotations:2.10.0"
```

```suggestion
    include "com.google.errorprone:error_prone_annotations:2.10.0"
```

to be safe; only <15kb